src/sage/coding/golay_code.py

```diff
@@ -178,7 +178,7 @@ def dual_code(self):
             return self
         return super().dual_code()
 
-    def minimum_distance(self):
```

Comment:
/home/runner/work/sage/sage/src/sage/coding/golay_code.py:181:35 - warning: Function with declared return type "int" must return value on all code paths
  Type "None" cannot be assigned to type "int" (reportGeneralTypeIssues)